<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OculusRiftPanel</class>
 <widget class="QWidget" name="OculusRiftPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="resetSensorButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reset Sensor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="trackingButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Oculus Rift Tracking (enabled)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="stereoPostButton">
     <property name="text">
      <string>Stereo and Post-Process on Client (enabled)</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Field of View:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FOV">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="fovSlider">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>300</number>
       </property>
       <property name="maximum">
        <number>1800</number>
       </property>
       <property name="value">
        <number>300</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="sensorPredictionButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Prediction</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pqComponents.qrc"/>
  <include location="../../../Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
